packages/mdc-textfield/_mixins.scss

```diff
@@ -20,35 +20,398 @@
 // THE SOFTWARE.
 //
 
+@import "@material/animation/variables";
 @import "@material/density/functions";
+@import "@material/feature-targeting/functions";
+@import "@material/feature-targeting/mixins";
 @import "@material/floating-label/mixins";
 @import "@material/line-ripple/mixins";
 @import "@material/notched-outline/mixins";
 @import "@material/notched-outline/variables";
-@import "@material/theme/mixins";
-@import "@material/shape/mixins";
+@import "@material/ripple/mixins";
+@import "@material/rtl/mixins";
 @import "@material/shape/functions";
+@import "@material/shape/mixins";
+@import "@material/theme/mixins";
+@import "@material/theme/variables";
+@import "@material/typography/mixins";
+@import "@material/typography/variables";
 @import "helper-text/mixins";
 @import "character-counter/mixins";
 @import "icon/mixins";
 @import "icon/variables";
 @import "./variables";
 @import "./functions";
 
+@mixin mdc-text-field-core-styles($query: mdc-feature-all()) {
+  @include mdc-text-field-without-ripple($query);
+  @include mdc-text-field-ripple($query);
+
+  // This needs to be included separately from `mdc-text-field-without-ripple` and it needs to come
+  // after `mdc-text-field-ripple` so that it can override the ripple styles and remove it.
+  @include mdc-text-field-outlined($query: $query);
```

Comment:
After re-ordering `-ripple()` and `-without-ripple()` mixins, I moved `mdc-text-field-outlined()` mixin to its original location. It fixes the issue that I described in [this comment](https://github.com/material-components/material-components-web/pull/5378#pullrequestreview-341461733).

Can we move this mixin after `.mdc-text-field__input:-webkit-autofill + .mdc-floating-label` selector block? This'll retain the CSS ordering for `__input` padding.